MODULE.bazel

```diff
@@ -3,18 +3,17 @@ module(
     version = "",
 )
 
+bazel_dep(
+    name = "abseil-cpp",
+    version = "20240116.0",
+    repo_name = "com_google_absl",
+)
```

Comment:
The extension was applying a patch. Any reason to change the behaviour? Is the patch part of `20240116.0`?